lib/inat_api_service/v2/client.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module INatAPIService
+  module V2
+    class Error < StandardError; end
```

Comment:
It didn't seem like this existed yet so let me know if:
1) You'd like to approach this in a different pattern or it isn't following how v2 should be consumed
2) You'd like to isolate to tooling to prevent premature use